src/multi-selector.html

```diff
@@ -5,6 +5,7 @@
 		<div class="cp-multi-selector__pill cps-white cps-bg-gray-10" ng-repeat="item in selectedItems | orderBy: 'display_name'">
 			{{item.display_name}}<i ng-click="removeItem(item)" class="cps-icon cps-icon-sm-close"></i>
 		</div>
+		<span ng-show="inputPlaceholder && (!selectedItems || !selectedItems.length)" class="cp-multi-selector__placeholder">{{inputPlaceholder}}</span>
```

Comment:
Valid question - the `<input>` is actually sneakily hidden behind this div to make this selector work. If you just put the placeholder on the input, it won't show.